artifactor/plugins/merkyl.py

```diff
@@ -71,6 +72,15 @@ def get_log(self, test_name, test_location, filename):
         content = doc.content
         return {"merkyl_content": content}, None
 
+    @ArtifactorBasePlugin.check_configured
+    def reset_log(self, test_name, test_location, filename):
+        test_ident = "{}/{}".format(test_location, test_name)
+        ip = self.tests[test_ident].ip
+
+        base, tail = os.path.split(filename)
```

Comment:
Required:
`base` not going to use in code please go with ` _, tail = os.path.split(filename)`